### PR TITLE
Run clang-format.

### PR DESCRIPTION
**Pull request by drwells**
I didn't rerun the test suite - its just clang-format. Lets see what the CI thinks.

### CODE REVIEW

**Comment by boyceg** [comment]
CI is not happy.

**Comment by drwells** [comment]
That's odd - the CI should be printing out a more detailed log.

File: examples/complex_fluids/ex2/InterpolationUtilities.h [hunk 1]
```diff
@@ -36,6 +33,10 @@
 
 #include <vector>
 
+#include <ibamr/app_namespaces.h>
+
+#include "ibamr/namespaces.h"
```
Comment (abarret) [inline examples/complex_fluids/ex2/InterpolationUtilities.h:38]: Do we need both `app_namespaces.h` and `namespaces.h`?

**Review by abarret — APPROVED** [review]

File: examples/complex_fluids/ex2/InterpolationUtilities.h (hunk 1 above)
Comment (drwells) [reply examples/complex_fluids/ex2/InterpolationUtilities.h:38]: No - I'll fix that.

**Pull request merged**